src/server/scripts/Outland/BlackTemple/boss_illidari_council.cpp

```diff
@@ -234,6 +244,20 @@ struct boss_illidari_council_memberAI : public ScriptedAI
         }
     }
 
+    void DamageTaken(Unit* /*attacker*/, uint32& damage, DamageEffectType /*damagetype*/, SpellSchoolMask /*damageSchoolMask*/) override
+    {
+        InstanceScript* instance = me->GetInstanceScript();
```

Comment:
Differently than spell scripts, here we don't need to validate instance script. That happens because the register method ("RegisterXXXXCreatureAI") ties the script to the instance itself, meaning the script will not run outside the designated instance

Reply:
Corrected.